packages/slate-editor/src/modules/editor/Editor.tsx

```diff
@@ -191,6 +192,35 @@ export const Editor = forwardRef<EditorRef, EditorProps>((props, forwardedRef) =
         }
     }, [autoFocus, editor]);
 
+    useEffect(() => {
+        function handleOutsideClick(event: MouseEvent) {
+            const clickTarget = event.target as HTMLElement | null;
+
+            if (!clickTarget) {
+                return;
+            }
+
+            function isTargetWithin(container: HTMLElement | null | undefined) {
+                return container && (container === clickTarget || container.contains(clickTarget));
+            }
+
+            const isWithinMenuPortal = isTargetWithin(popperMenuOptions.portalNode?.current);
+            const isWithinEditor = isTargetWithin(containerRef.current);
+
+            if (!isWithinEditor && !isWithinMenuPortal) {
+                EditorCommands.blur(editor);
+            }
+        }
+
+        if (loseFocusOnOutsideClick) {
```

Comment:
Hmm, if only there was a word for the opposite of focus 😋 How about.. `blurOnOutsideClick`? 😄

Reply:
Right! That's much better, thanks Lukas! ❤️